### PR TITLE
gl-1383 move GDCWholeGenomeSomaticSingleSample.wdl out of beta pipelines and remove miniwdl check

### PR DESCRIPTION
**Pull request by nikellepetrillo**
moved GDCWholeGenomeSomaticSingleSample.wdl out of beta into pipelines, updated changelog/version, updated paths in scala test/pullapprove template

### CODE REVIEW

**Review by jessicaway — APPROVED** [review]
Sorry to totally throw a wrench in things (also you can disagree and merge anyways), but I would have put these in `/warp/pipelines/broad/dna_seq/somatic/...`

**Comment by nikellepetrillo** [comment]
retest this please

File: pipelines/broad/dna_seq/somatic/single_sample/wgs/gdc_genome/GDCWholeGenomeSomaticSingleSample.changelog.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# 1.0.1
+2020-03-02
```
Comment (gbggrant) [inline pipelines/broad/dna_seq/somatic/single_sample/wgs/gdc_genome/GDCWholeGenomeSomaticSingleSample.changelog.md:2]: 2021-03-02!

**Review by gbggrant — APPROVED** [review]
Looks good.  One small fix in date.

File: pipelines/broad/dna_seq/somatic/single_sample/wgs/gdc_genome/GDCWholeGenomeSomaticSingleSample.changelog.md (hunk 1 above)
Comment (nikellepetrillo) [reply pipelines/broad/dna_seq/somatic/single_sample/wgs/gdc_genome/GDCWholeGenomeSomaticSingleSample.changelog.md:2]: Ugh! Good catch!

**Pull request merged**